.idea/workspace.xml

```diff
@@ -2,8 +2,55 @@
 <project version="4">
```

Comment:
We should really find a way to remove this Idea folder from the repo. It's huge bloat, adding unneeded files and slows down code review.